_config.yml

```
@@ -1,96 +1,37 @@
-# Site settings
-title: "The Union of UK Unicyclists"
-email: webmaster@unicycle.org.uk
-description: > # this means to ignore newlines until "baseurl:"
-  The website of the Union of UK Unicyclists
-baseurl: "" # the subpath of your site, e.g. /blog/
-url: "http://www.unicycle.org.uk" # the base hostname & protocol for your site
-twitter_username: UUUnicyclists
-github_username:  unionofukunicyclists
-facebook_page: theunionofukunicyclists
-google_maps_api_key: "AIzaSyBoPKA9DXHMbdvaqMPWxXGX1rUbAJN5JsI"
+# site settings
+title: The Union of UK Unicyclists
+shorttitle: UUU
+subtitle: The Union of UK Unicyclists
+email: contact@unicycling.org.uk
```

This is not a proper e-mail address. I've added you, Edd, and Roger to `committee@unicycle.org.uk` so we should just use that.

```suggestion
email: committee@unicycle.org.uk
```